samples/unit_scaling.exs

```diff
@@ -36,17 +36,17 @@ Benchee.run %{
 # With :smallest scaling
 # Name           ips        average    deviation         median
 # 10      3398103.02        0.29 μs    (±20.77%)        0.29 μs
-# 100      196741.99        5.08 μs    (±13.06%)        5.00 μs
-# 1k         8341.69      119.88 μs     (±8.84%)      118.00 μs
-# 10k         577.54     1731.49 μs     (±9.98%)     1693.00 μs
+# 100      196741.99        5.08 μs    (±13.06%)        5 μs
+# 1k         8341.69      119.88 μs     (±8.84%)      118 μs
+# 10k         577.54     1731.49 μs     (±9.98%)     1693 μs
 # 100k         42.87    23326.77 μs    (±12.01%)    21974.50 μs
 # 1M            3.27   305768.81 μs     (±3.05%)   306229.50 μs
 
 # With :none scaling
 # Name           ips        average    deviation         median
 # 10      3587943.97        0.28 μs    (±21.22%)        0.27 μs
 # 100      201976.59        4.95 μs    (±14.42%)        4.80 μs
-# 1k         8480.32      117.92 μs     (±8.46%)      117.00 μs
-# 10k         560.89     1782.89 μs    (±10.70%)     1742.00 μs
+# 1k         8480.32      117.92 μs     (±8.46%)      117 μs
+# 10k         560.89     1782.89 μs    (±10.70%)     1742 μs
```

Comment:
This seems like a small thing, and it might just be from the search and replace, but the `μs` are no longer vertically aligned in this sample as I'd expect them to be.

It also looks like the formatting is a little off in the other examples as well - `μs` is no longer right aligned with the "n" in `median`.

This might not be worth fixing, but it just jumped out to me.